docs/source/usage/cli.rst

```diff
@@ -170,11 +170,13 @@ Image
 
 Image-based provisioning can be accomplished through the use of `Metalsmith`_. It requires the image to be uploaded into OpenStack Glance. Once that's done, a non-admin can run the following:
 
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-|                | **Actions**                                                                                                         |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>`` |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
++----------------+---------------------------------------------------------------------------------------------------------------------------------------------+
+|                | **Actions**                                                                                                                                 |
++----------------+---------------------------------------------------------------------------------------------------------------------------------------------+
+| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --candidate <node-uuid> --network <network> --ssh-public-key <path-to-key>`` |
```

Comment:
I'm pretty sure that the node name works here as well, so <node id> could be used here

Reply:
Here also I will mention <node id> instead